win32ss/gdi/ntgdi/font.c

```diff
@@ -749,6 +749,7 @@ NtGdiGetGlyphOutline(
         EngSetLastError(ERROR_NOT_ENOUGH_MEMORY);
         goto Exit;
      }
+     RtlZeroMemory(pvBuf, cjBuf);
```

Comment:
Let's use `ExAllocatePoolZero` since we have it in our sdk now

Reply:
Thanks for your advice, I have changed that function.